### DISCUSSION TITLE
View Data Shared by Copilot

### DISCUSSION THREAD

**Discussion by drexalt** [community/community#36649] [Copilot Conversations] [answered]
Is there a way to view the data Copilot is sharing? I know you can select telemetry settings, is there a way to view/track the data that was shared? 

Thanks!

**Comment by cmp-nct** [comment]
I'd also like to know how much data was transmitted, how much of it is still stored and have a control over it.
Proprietary code sent to copilot needs to be auto-removed after a while

**Comment by ossanna16** [comment]
Hey there @jturner116 and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by drexalt** [reply]
Thanks @ossanna16 ! Is there anyone I could ping to learn more? I've looked for other mentions of how to view what data is transmitted, but I haven't found anything solid.

Thanks!!

**Reply by ossanna16** [reply]
Hey there! No one I could specifically guide you to but our Codepilot team is keeping an eye on this space and will hopefully be in touch with you!

**Comment by iamcjwilliams** [comment, marked as answer]
@jturner116 Currently there's no way to view data, I recommend that you keep an eye on our [official GitHub public product roadmap](https://github.com/github/roadmap), our [Changelog](https://github.blog/changelog/) and our [GitHub Blog](http://github.blog/) for future updates.

**Reply by drexalt** [reply]
@iamcjwilliams I was keeping an eye on your official roadmap and noticed copilot oddly dropped off of the roadmap entirely. Any updates you can give before I send my $10? Lol

**Comment by tjerkw** [comment]
I disabled sharing code snippets but the AI still understands my code, is my code somehow STILL shared with microsoft over the network?
